database/connect.ts

```diff
@@ -1,50 +1,28 @@
-import 'server-only';
-import { config } from 'dotenv-safe';
-import { headers } from 'next/headers';
-import postgres, { Sql } from 'postgres';
+import postgres from 'postgres';
+import { setEnvironmentVariables } from '../ley.config.mjs';
 
-if (!process.env.FLY_IO) config();
+// This loads all environment variables from a .env file
+// for all code after this line
+setEnvironmentVariables();
 
+// Type needed for the connection function below
 declare module globalThis {
-  let postgresSqlClient: Sql;
+  let postgresSqlClient: ReturnType<typeof postgres> | undefined;
```

Comment:
I think `Sql` is the new version, not `ReturnType<...>`

Reply:
let postgresSqlClient: Sql;